src/gui/addnewtorrentdialog.h

```diff
@@ -51,15 +51,14 @@ namespace Ui
 class PropListDelegate;
 class TorrentContentFilterModel;
 class TorrentFileGuard;
-template <typename T> class CachedSettingValue;
```

Comment:
You also need remove appropriate include from .cpp, isn't it?

Reply:
done.